src/main/target/BEEBRAIN_V2F/config.c

```diff
@@ -132,6 +132,7 @@ void targetConfiguration(void)
     osdConfigMutable()->item_pos[OSD_GPS_SATS]           &= ~VISIBLE_FLAG;
     osdConfigMutable()->item_pos[OSD_HOME_DIR]           &= ~VISIBLE_FLAG;
     osdConfigMutable()->item_pos[OSD_HOME_DIST]          &= ~VISIBLE_FLAG;
+    osdConfigMutable()->item_pos[OSD_TOTAL_DIST]         &= ~VISIBLE_FLAG;
```

Comment:
It should be left to target maintainers to make these changes - adding not explicitly required code here just adds to the maintenance effort for targets.

Reply:
done